src/fileiter.cpp

```diff
@@ -53,6 +53,12 @@ namespace std{
 #include <sys/cygwin.h>
 #endif
 
+#if defined (BOOST_ASIO_WINDOWS_RUNTIME) || defined (BOOST_PLAT_WINDOWS_PHONE)
```

Comment:
@jzmaddock Thanks for the kind words!  From responses I received from the Boost mailing list, I am not aware of being able to run the Boost regression tests for Windows Phone.  I will say that I have been using my fixes, originally done on version 1.48, for the past few months with no problems at all, but that will not convince you or anyone else they are good :-).